src/test/analyzer_test.ts

```diff
@@ -47,16 +48,155 @@ suite('Analyzer', () => {
     });
   });
 
-
   suite('analyze()', () => {
 
+    test(
+        'analyzes a document with an inline Polymer element feature',
+        async() => {
+          const document = await analyzer.analyze(
+              'static/analysis/simple/simple-element.html');
+          const elements = document.getByKind('element');
```

Comment:
This will give you *much* better error messages if this test ever breaks:

```typescript
const elements = Array.from(document.getByKind('element'));
assert.deepEqual(elements.map(e => e.tagName), ['simple-element']);
```

Asserting on size and then on individual items means that if there are e.g. two elements that are returned, all you find out is that there are two, not _which_ two. The first thing you'll want to do then is to go and add some console.logs or whatever to tell you that info.

With the above construct you'll get a nice diff like:

```diff
    [
      'simple-element',
+     'other-element',
    ]
```